### PR TITLE
Fix(Qualifications): replace user role with user qualifications

### PR DESCRIPTION
**Pull request by stavbi1**
User qualifications is the הכשרת בסיס of the person:
    ('תוכניתן'),
    ('בודק תוכנה'),
    ('כותב לומדה'),
    ('מיישם'),
    ('אנליסט'),
    ('אחר');

user roles are the jobs he worked in through the years:
    ('תוכניתן')
    ('רש"צ')
    ('מנהל מוצר')
    ('קצין פרויקט')
    ('ארכיטקט')

### CODE REVIEW

File: scripts/DDL.sql [hunk 1]
```diff
@@ -12,6 +12,11 @@ CREATE TABLE jobs.roles(
 	color text
 );
 
+CREATE TABLE jobs.qualifications(
+	id serial PRIMARY KEY,
+	name text UNIQUE
```
Comment (guyshany) [inline scripts/DDL.sql:17]: Not sure about the exact flow, but why don't we give the qualifications an english identifier that will be more readable then a random number?  
 
Is there a specific reason (in application flow terms) for making it a sequence?

File: src/components/Personal-Zone/Searcher/AddResumeDialog/ProgressBar/ProgressBar.tsx [hunk 2]
```diff
@@ -29,7 +29,8 @@ const ProgressBar: React.FC<ProgressBarProps> = (props): JSX.Element => {
     const { allSelectOptions } = props;
 
     const [activeStep, setActiveStep] = React.useState<ResumeStep>(ResumeStep.PERSONAL_DETAILS);
-    const [selectedRoleId, setSelectedRoleId] = useState<number>(allSelectOptions?.roleOptions[0].id || 0);
+    const [selectedQualificationId, setSelectedQualificationId] =
+        useState<number>(allSelectOptions?.qualificationOptions[0].id || 0);
```
Comment (guyshany) [inline src/components/Personal-Zone/Searcher/AddResumeDialog/ProgressBar/ProgressBar.tsx:33]: `allSelectOptions?.qualificationOptions[0].id` - Nice!! didn't know this syntax

File: src/types/Qualification.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import { SelectChooseOption } from "./ChooseOption";
```
Comment (guyshany) [inline src/types/Qualification.ts:1]: Whats the purpos of this file? 

If we will have types in the future, we can always create a qualification types file

File: scripts/DDL.sql (hunk 1 above)
Comment (stavbi1) [reply scripts/DDL.sql:17]: yes, a sequence is a better practice for indexing

File: src/types/Qualification.ts (hunk 3 above)
Comment (stavbi1) [reply src/types/Qualification.ts:1]: we have types now, the future is here

**Pull request merged**